src/Misc/MiddleWare.cpp

```diff
@@ -1592,6 +1654,14 @@ static rtosc::Ports middwareSnoopPortsWithoutNonRtParams = {
          std::string url = rtosc_argument(msg, 0).s;
          void* obj = impl->obj_store.get(url);
 
+         {
+             int part, kit, vc;
```

Comment:
I'd give these values some initialized value (zeros) just to be safe.

Reply:
The implementation of `idsFromMsg` always returns false if a passed parameter (like `part`, `kit`, `vc`) could not be filled. This is a design goal of that function, and it's also tested in src/Tests/MsgParseTests.h. So I really want to avoid blowing up the code with unnecessary initializations.